dbt/include/spark/macros/materializations/seed.sql

```diff
@@ -81,10 +81,7 @@
   {%- set agate_table = load_agate_table() -%}
   {%- do store_result('agate_table', response='OK', agate_table=agate_table) -%}
 
-  {{ run_hooks(pre_hooks, inside_transaction=False) }}
```

Comment:
Why do the `run_hooks` call change? Is it related to the `persist_docs` change or is this related to something else?

Reply:
This isn't really related to the specific change in the PR at all, I just noticed because I was editing related lines. We clearly copied these lines from the default `seed` materialization a few years back. The default materialization, which was originally written for Redshift/Postgres, assumes a transactional database. Some hooks may want to run before the main transaction starts (`begin`), after it starts, before it ends (`commit`), or after it ends.

Spark doesn't have transactions, so there's no need to distinguish between `inside_transaction = True|False`. I figured I would simplify it while here, but I'd also be fine reverting those lines for PR cleanliness.